vlog.go

```diff
@@ -44,7 +48,7 @@ func openValueLog(options valueLogOptions) (*valueLog, error) {
 		vLogWal, err := wal.Open(wal.Options{
 			DirPath:        options.dirPath,
 			SegmentSize:    options.segmentSize,
-			SegmentFileExt: fmt.Sprintf(valueLogFileExt, i),
+			SegmentFileExt: fmt.Sprintf(valueLogFileExt, time.Now().Format("02-03-04-05-2006"), i),
```

Comment:
Can we add a suffix like `.temp`, after compaction, we can erase the suffix name to make it as a normal WAL file.

Reply:
> Can we add a suffix like `.temp`, after compaction, we can erase the suffix name to make it as a normal WAL file.

I have considered this method, but in this way, maybe we need to add a rename fnnction in WAL. If we use time stamp to build wal file, there are not need to rename after compaction.